scripts/merge_users.py

```diff
@@ -0,0 +1,285 @@
+@click.option('-db', '--mongodb', help='Name of mongo database [scout]')
+@click.option('-u', '--username')
+@click.option('-p', '--password')
+@click.option('-a', '--authdb', help='database to use for authentication')
+@click.option('-port', '--port', help="Specify on what port to listen for the mongod")
+@click.option('-h', '--host', help="Specify the host for the mongo database.")
+@click.option('-c', '--config', type=click.Path(exists=True),
+              help="Specify the path to a config file with database info.")
+@click.option('-L', '--live', is_flag=True, help='live run - use with extreme caution')
+@click.version_option(__version__)
+@click.pass_context
+def merge_users(context, mongodb, username, password, authdb, host, port, loglevel, config, live):
+    """scout: manage interactions with a scout instance."""
+    coloredlogs.install(level=loglevel)
+
+    LOG.info("Running scout version %s", __version__)
+    LOG.debug("Debug logging enabled.")
+
+    mongo_config = {}
+    cli_config = {}
+    if config:
+        LOG.debug("Use config file %s", config)
+        with open(config, 'r') as in_handle:
+            cli_config = yaml.load(in_handle, Loader=yaml.FullLoader)
+
+    mongo_config['mongodb'] = (mongodb or cli_config.get('mongodb') or 'scout')
+
+    mongo_config['host'] = (host or cli_config.get('host') or 'localhost')
+    mongo_config['port'] = (port or cli_config.get('port') or 27017)
+    mongo_config['username'] = username or cli_config.get('username')
+    mongo_config['password'] = password or cli_config.get('password')
+    mongo_config['authdb'] = authdb or cli_config.get('authdb') or mongo_config['mongodb']
+    mongo_config['omim_api_key'] = cli_config.get('omim_api_key')
+
+
+    # always dryrun for now
+    dryrun = True
+    if live:
+        dryrun = False
+
+    LOG.info("Setting database name to %s", mongo_config['mongodb'])
+    LOG.debug("Setting host to %s", mongo_config['host'])
+    LOG.debug("Setting port to %s", mongo_config['port'])
+
+    valid_connection = check_connection(
+        host=mongo_config['host'],
+        port=mongo_config['port'],
+        username=mongo_config['username'],
+        password=mongo_config['password'],
+        authdb=mongo_config['authdb'],
+    )
+
+    LOG.info("Test if mongod is running")
+    if not valid_connection:
+        LOG.warning("Connection could not be established")
+        context.abort()
+
+    try:
+        client = get_connection(**mongo_config)
+    except ConnectionFailure:
+        context.abort()
+
+    database = client[mongo_config['mongodb']]
+
+    LOG.info("Setting up a mongo adapter")
+    mongo_config['client'] = client
+    adapter = MongoAdapter(database)
+
+    user_requests = []
+    event_requests = []
+    acmg_requests = []
+    clinvar_requests = []
+    clinvar_submission_requests = []
+
+    for oi_user_obj in adapter.user_collection.find({'_id': {'$type':'objectId'}}):
+        if not oi_user_obj.get('email'):
```

Comment:
Yes, at least one; but that one does not seem to connect to any events fortunately. It seems it would have been a little hard to log in with any decently modern version of the codebase, so probably predates events.